src/libpsl-native/src/nativesyslog.cpp

```diff
@@ -0,0 +1,17 @@
+#include <syslog.h>
+#include <nativesyslog.h>
+
+extern "C" void Native_SysLog(int32_t priority, const char* message)
+{
+    syslog(priority, "%s", message);
+}
+
+extern "C" void Native_OpenLog(const char* ident, int facility)
```

Comment:
Since Native_OpenLog takes a const char* it shouldn't need special marshaling.  But I assume you do this above because it keeps a reference to the ident parameter string?  If so can you update the comment to make this clear and as to why FreeHGlobal is never called (presumably because the allocation needs to exist for the life of the process).